src/index.ts

```diff
@@ -104,6 +105,16 @@ export class WebRTCPlayer extends EventEmitter {
     }
   }
 
+  private onConnectionStats() {
+    if (this.peer) {
+      this.peer.getStats(null).then((stats) => {
+        stats.forEach((report) => {
+          this.emit(`stats:${report.type}`, report);
```

Comment:
This will be more than 100 emitted events per stats object. Could it be done so that only interesting aggregate stats are presented?

Reply:
Oh, that much? Ok yes, will implement a way for a client to subscribe to only the ones relevant